templates/definition/meter/goodwe-hybrid.yaml

```diff
@@ -0,0 +1,100 @@
+template: goodwe-hybrid
+products:
+  - brand: Goodwe
+    description:
+      generic: ET/EH/BH/BT Hybrid Inverter
+params:
+  - name: usage
+    choice: ["grid", "pv", "battery"]
+  - name: modbus
+    choice: ["rs485", "tcpip"]
+    baudrate: 9600
+    id: 247
+  - name: capacity
+    advanced: true
+render: |
+  type: custom
+  power:
```

Comment:
schöner: den Wert in jede Usage mit rein, ist übersichtliche und leichter zu lesen

Reply:
Stimmt. Ist aber bei vielen anderen Templates auch so.

Reply:
Yep, sollten wir trotzdem nach und nach ändern.